dotnet/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion>11</LangVersion>
+    <LangVersion>latest</LangVersion>

```

Comment:
What is the benefit of using `latest` over a specific version? I'd rather not see the build break when a new language version is introduced.

Reply:
I will do it later